heft-plugins/heft-webpack4-plugin/package.json

```diff
@@ -18,11 +18,11 @@
     "_phase:test": "heft test --no-build"
   },
   "peerDependencies": {
-    "@rushstack/heft": "^0.44.2"
+    "@rushstack/heft": "^0.44.2",
+    "webpack": "~4.44.2"
```

Comment:
```suggestion
    "webpack": "^4.44.2"
```

Reply:
@iclanton  Webpack and its dependencies tend to have a lot of breakage on upgrades.  The existing rigs used `~` versioning, so I'm hesitant to change that policy.